.gitignore

```diff
@@ -40,6 +40,7 @@ wp-tests-config.php
 /packagehash.txt
 /artifacts
 /setup.log
+/.idea
```

Comment:
I guess this is an unwanted change :)

Reply:
Reverted it. But what do we gotta do with the problem then:
IDE PhpStorm (by IntellijIdea) creates .idea/ folder at the repository root, so either I'm ignoring that or selecting files to commit one-by-one which is kinda unhandy